content/intro-to-storybook/ember/en/screen.md

```diff
@@ -0,0 +1,201 @@
+  { id: '3', title: 'Something else', state: 'TASK_INBOX' },
+  { id: '4', title: 'Something again', state: 'TASK_INBOX' },
+];
+
+const initialState = {
+  isError: false,
+  tasks: defaultTasks,
+};
+// The actions are the "names" of the changes that can happen to the store
+export const actions = {
+  ARCHIVE_TASK: 'ARCHIVE_TASK',
+  PIN_TASK: 'PIN_TASK',
+  SET_ERROR: 'SET_ERROR',
+};
+
+// The action creators bundle actions with the data required to execute them
+export const archiveTask = id => ({ type: actions.ARCHIVE_TASK, id });
+export const pinTask = id => ({ type: actions.PIN_TASK, id });
+export const setError = () => ({ type: actions.SET_ERROR });
+// All our reducers simply change the state of a single task.
+function taskStateReducer(taskState) {
+  return (state, action) => {
+    return {
+      ...state,
+      tasks: state.tasks.map(task =>
+        task.id === action.id ? { ...task, state: taskState } : task
+      ),
+    };
+  };
+}
+
+// The reducer describes how the contents of the store change for each action
+export const reducer = (state, action) => {
+  switch (action.type) {
+    case actions.ARCHIVE_TASK:
+      return taskStateReducer('TASK_ARCHIVED')(state, action);
+    case actions.PIN_TASK:
+      return taskStateReducer('TASK_PINNED')(state, action);
+    case actions.SET_ERROR:
+      return {
+        ...state,
+        isError: true,
+      };
+    default:
+      return state || initialState;
+  }
+};
+
+export default combineReducers({
+  reducer,
+});
+```
+
+The store is updated with the new field. Let's create a presentational `PureInboxScreen.hbs` in our `app/components` folder:
+
+```handlebars
+{{!--app/components/PureInboxScreen.hbs--}}
+{{#if @error}}
+  <div class="page lists-show">
+    <div class="wrapper-message">
+        <span class="icon-face-sad" />
+        <div class="title-message">Oh no!</div>
+        <div class="subtitle-message">Something went wrong</div>
+    </div>
+  </div>
+{{else}}
+  <div class="page lists-show">
+    <nav>
+        <h1 class="title-page">
+            <span class="title-wrapper">Taskbox</span>
+        </h1>
+    </nav>
+    {{TaskList}}
+  </div>
+{{/if}}
+```
+
+Then, we can create a container, which again grabs the data for the `PureInboxScreen` in `app/components/InboxScreen.hbs` add the following:
+
+```handlebars
+{{!--app/components/InboxScreen.hbs--}}
+<div>
+    {{PureInboxScreen error=error}}
+</div>
+```
+
+In `app/components/InboxScreen.js` add the following:
+
+```javascript
+// app/components/InboxScreen.js
+import Component from '@glimmer/component';
+import { connect } from 'ember-redux';
+import { computed } from '@ember/object';
+
+const stateToComputed = state => {
+  const { reducer } = state;
+  const { isError } = reducer;
+
+  return {
+    error: isError,
+  };
+};
+class InboxScreen extends Component {
+  constructor() {
+    super(...arguments);
+  }
+  @computed('error')
+  get error() {
+    return this.error;
+  }
```

Comment:
I think neither of this is needed. But I don't use redux, so there might be a need for the `error`, then do this:

```suggestion
  get error() {
    return this.args.error;
  }
```

It's autotracked already and will re-evaluate on access: https://www.pzuraq.com/how-autotracking-works/ (If you cannot access `this.args.error` directly from redux. If it is needed on template level, then just `@error`.